plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java

```diff
@@ -139,6 +142,13 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
         };
     }
 
+    @Test
+    @Override
+    public void testUpdateRowConcurrently()
+    {
+        abort("Postgresql doesn't support concurrent update of different columns in a row");
```

Comment:
Can you do something like:
```
        assertThatThrownBy(super:: testUpdateRowConcurrently).hasMessageContaining("....");
```

Reply:
It fails due to the incorrect result, and the result usually is non deterministic.

Reply:
Tried, PTAL

Reply:
> It fails due to the incorrect result

Oh... It is a big deal unfortunately. It is ok to fail with some nice error message, however no failure and incorrect results are not good.

As I understand now merge is being done on many worker nodes. Is it possible to do it on one worker only and do `commit` or `rollback` so either all UPDATE is done or none?

Reply:
Interesting, do u have some context about this, I mean which part I should learn for and refactor?

Reply:
I think you are talking is similar to the FTE, which would write the `update` and `delete` into temporary table as well

Reply:
How would it work?

Reply:
link https://github.com/trinodb/trino/pull/19819. Here is a workaround.

Reply:
Will be resolved by https://github.com/trinodb/trino/pull/24075